app/helpers/typography_helper.rb

```diff
@@ -0,0 +1,5 @@
+module TypographyHelper
+  def nbsp_between_last_two_words(text)
+    text.sub(/\s([\w\.\?\!]+)$/, '&nbsp;\1').html_safe
```

Comment:
It would be good to have a test that would assert what happens if the `text` input included markup, eg:
```
assert_equal "this &lt;b&gt;&nbsp;&lt;/b&gt; that", nbsp_between_last_two_words("this <b>and</b> that")
```

I think there's a bug here, even if not, it would be good for the test to cover that case if the implementation changed, and to self-document.